mne/viz/tests/test_topomap.py

```diff
@@ -728,3 +729,81 @@ def test_plot_bridged_electrodes():
 
     with pytest.raises(RuntimeError, match='Expected'):
         plot_bridged_electrodes(info, bridged_idx, np.zeros((5, 6, 7)))
+
+
+def test_plot_ch_adjacency():
+    """Test plotting of adjacency matrix."""
+    xyz_pos = np.array([[-0.1, 0.1, 0.1], [0.1, 0.1, 0.1], [0., 0., 0.12],
+                        [-0.1, -0.1, 0.1], [0.1, -0.1, 0.1]])
+
+    info = create_info(list('abcde'), 23, ch_types='eeg')
+    montage = make_dig_montage(
+        ch_pos={ch: pos for ch, pos in zip(info.ch_names, xyz_pos)},
+        coord_frame='head')
+    info.set_montage(montage)
+
+    # construct adjacency
+    adj, ch_names = find_ch_adjacency(info, 'eeg')
+    adj = adj.toarray()
+
+    # plot adjacency
+    fig = plot_ch_adjacency(info, adj, ch_names, kind='2d', edit=True)
```

Comment:
looks like you never call with `kind='3d'`, just doing this as a "smoke test" would be good, I think.

This would also improve coverage because your new `set_3d_axes_equal` would be called under the hood.

Reply:
This would also improve coverage because your new `set_3d_axes_equal` would be called under the hood.

Reply:
Ah I see this is still on your to do list! ... as well as including some tests that raise the expected errors, great!